WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -125,9 +125,9 @@ extension NSNotification.Name {
         helpCenterConfig.labels = [Constants.articleLabel]
 
         // If we don't have the user's information, disable 'Contact Us' via the Help Center and Article view.
-        helpCenterConfig.hideContactSupport = !haveUserIdentity
+        helpCenterConfig.showContactOptionsOnEmptySearch = haveUserIdentity
```

Comment:
This isn't working as expected. If I have not entered my contact information, when I go to the Help Center, the Contact Us button is available in the nav bar (it should not be):

<img width="400" alt="help_center" src="https://user-images.githubusercontent.com/1816888/62563697-c855e180-b840-11e9-9f14-f4bcb4430cf4.png">

I think it's just the wrong method. I changed it to `showContactOptions` and it worked fine.